lib/util/player_configuration.js

```diff
@@ -431,6 +431,7 @@ shaka.util.PlayerConfiguration = class {
       },
       dispatchAllEmsgBoxes: false,
       useSourceElements: true,
+      durationReductionEmitsUpdateEnd: true,
```

Comment:
This seems fine to me.  If there are specific platforms that need this set to `false`, you can follow up with platform detection that sets the flag to `false` here on certain platforms.